test/ng/rootScopeSpec.js

```diff
@@ -13,9 +13,7 @@ describe('Scope', function() {
 
 
     it('should expose the constructor', inject(function($rootScope) {
-      if (msie < 11) return;
-      // eslint-disable-next-line no-proto
-      expect($rootScope.__proto__).toBe($rootScope.constructor.prototype);
+      expect(Object.getPrototypeOf($rootScope)).toBe($rootScope.constructor.prototype);
```

Comment:
It was there only because of the Annex B (i.e. compatibility "don't use" stuff) `__proto__` that wasn't present in IE <11. The standard ES5+ way to get the prototype is `Object.getPrototypeOf(object)` which works in IE9+.